docs/projects/cn0561/index.rst

```diff
@@ -0,0 +1,280 @@
+.. _cn0561:
+
+CN0561 HDL project
+===============================================================================
+
+Overview
+-------------------------------------------------------------------------------
+
+The :adi:`AD4134` is a quad channel, low noise, simultaneous sampling, precision 
+analog-to-digital converter (ADC), based on the continuous time sigma-delta 
+(CTSD) modulation scheme. This architecture inherently rejects signals around
+the ADC aliasing frequency band, giving the device its inherent antialiasing 
+capability, and removesthe need for a complex external antialiasing filter.
+
+This device has four independent converter channels in parallel, each with a CTSD 
+modulator and a digital decimation and filtering path. It enables simultaneous 
+sampling of four signal sources, with a maximum input bandwidth of 391.5 kHz.
+It supports a wide range of ODR frequencies, from 0.01 kSPS to 1496 kSPS wih less
+than 0.01 SPS adjustment resolution, allowing the user to granularly vary sampling
+speed to achieve coherent sampling.
+
+The :adi:`AD4134` supports two device configuration schemes: serial peripheral interface
+(SPI) and hardware pin configuration (pin control mode). The SPI control mode offers
+access to all the features and configuration options available on the chip.
+Pin control mode offers the benefit of simplifying the device configuration, enabling
+the device to operate autonomously after power-up operating in a standalone mode.
+
+The HDL reference design for the EVAL-CN0561 provides all the interfaces that are 
+necessary to interact with the device using a Xilinx FPGA development board; to acquire 
+data from the ADC device, supporting continuous data capture at maximum 1.5 MSPS data rate
+However, due to a hardware limitation, the Cora-Z7s variant will only support a maximum 
+data clock of 24MHz in contrast with 48MHz supported on the Zedboard.
+
+Supported boards
+-------------------------------------------------------------------------------
+
+-  :adi:`EVAL-CN0561 <EVAL-AD4134FMCZ>` *
+
+.. admonition:: Legend
+   :class: note
+
+   ``*`` Evaluation board page currently unavailable. See EVAL-AD4134FMCZ board, similar hardware, using the same AD4134 adc device.
```

Comment:
Please wrap the text to fit within 80 characters/line.

Reply:
I think there shouldn't be a link to a page that doesn't exist yet (lines 37-42).
It is better to not be specified and then be added, rather than saying there isn't and then edit it anyway to remove the note.